pytorch_lightning/loggers/base.py

```diff
@@ -297,6 +297,20 @@ def log_graph(self, model: "pl.LightningModule", input_array=None) -> None:
         """
         pass
 
+    def log_text(self, *args, **kwargs) -> None:
+        """Log text.
+
+        Arguments are directly passed to the logger.
```

Comment:
the docstring is somewhat confusing. won't people be calling `SomeLoggerImplementation.log_text` ?

Reply:
It is actually working like the other methods so we can call the method through the logger or through the lightning module.